src/tags/strikethrough/index.js

```diff
@@ -1,8 +1,8 @@
 class Bold {
-  constructor (quillJS) {
+  constructor (quillJS, options = {}) {
     this.quillJS = quillJS
     this.name = 'strikethrough'
-    this.pattern = /(?:~|_){2}(.+?)(?:~|_){2}/g
+    this.pattern =  options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g
```

Comment:
If you want to pass test case, you need to replace multiple white space to single white space.

Mean:
```
this.pattern =..options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g

to

this.pattern =.options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g

```

Dot is whitespace mean

Reply:
Thank you. I fixed it.

Reply:
Thank you. I fixed it.

Reply:
AWESOME!